src/controllers/brands/put.controller.ts

```diff
@@ -4,9 +4,10 @@ import { body } from 'express-validator';
 import BrandService from '../../services/BrandService';
 
 export default {
-    validation: [body('logoImgUrl').exists().isURL(), body('backgroundImgUrl').exists().isURL()],
+    validation: [body('logoImgUrl').optional().isURL(), body('backgroundImgUrl').optional().isURL()],
     controller: async (req: Request, res: Response) => {
-        const brand = await BrandService.update({ poolId: req.params.poolId }, req.body);
+        console.log(req.assetPool._id);
```

Comment:
@GarfDev Cleanup! I will fix this one though. We also require support for this when QR codes are generated in the backend as part of a bulk operation. It's mentioned in the issue, but I see the reference to the original comment in the PR is broken. This was the comment meant for it:
https://github.com/thxprotocol/api/pull/360#discussion_r924374171

I will add a ticket to add support for this during sprint 5